### PR TITLE
Tag Automa.jl v0.2.4

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/Automa.jl](https://github.com/BioJulia/Automa.jl)
Release: [v0.2.4](https://github.com/BioJulia/Automa.jl/releases/tag/v0.2.4)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/Automa.jl/builds/237751161)
Diff: [vs v0.2.3](https://github.com/BioJulia/Automa.jl/compare/1f90c01750e1b1c89ca8e99af71f1ba0cb094772...3783273b83ec5ed7c16676ef8036df4bc1c59bf5)
`requires` vs v0.2.3: no changes
cc: @bicycle1885

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by bicycle1885** [comment]
I made a mistake, please hold on for a moment.

**Comment by bicycle1885** [comment]
Okay, it gets ready now.

File: Automa/versions/0.2.4/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+1f90c01750e1b1c89ca8e99af71f1ba0cb094772
```
Comment (tkelman) [inline Automa/versions/0.2.4/sha1]: this is the same commit as 0.2.3

Comment (bicycle1885) [inline Automa/versions/0.2.4/sha1]: Oops, sorry. Updated.

**Pull request merged**